manifests/Nuclear Missile Buff.yaml

```
@@ -0,0 +1,13 @@
+name: Nuclear Missile Buff
+authors: BZENDSKY
+homepage: https://github.com/BZENDSKY/Nuclear-Missile-Buff
+license: GPL-3.0-or-later
+version: v1.0
+shortDescription: Buffs the Nuclear Missile.
+description: Gives the Nuclear Missile more shield and hull damage.
+url: https://github.com/BZENDSKY/Nuclear-Missile-Buff/archive/refs/tags/v1.0.zip
+iconUrl: https://github.com/BZENDSKY/Nuclear-Missile-Buff/raw/nuclear/icon.png
```

Version this one as well, please:
```suggestion
iconUrl: https://github.com/BZENDSKY/Nuclear-Missile-Buff/raw/v1.0/icon.png
```